<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Neural_network_card</class>
 <widget class="QFrame" name="Neural_network_card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="card">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QFrame{
background-image: url(:/picture/card.png);
background-repeat: no-repeat;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="clustering_rate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QLabel" name="positive_cr">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>250</y>
      <width>71</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>?/?</string>
    </property>
   </widget>
   <widget class="QLabel" name="negative_cr">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>270</y>
      <width>71</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>?/?</string>
    </property>
   </widget>
   <widget class="QLabel" name="cr">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>290</y>
      <width>71</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>?/?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>positive :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>negative :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}
</string>
    </property>
    <property name="text">
     <string>total :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>neurons in hidden layers :</string>
    </property>
   </widget>
   <widget class="QLabel" name="number_of_neurons_in_hidden_layer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>95</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>learning rate :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : black;
}</string>
    </property>
    <property name="text">
     <string>batch size :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="batch_size">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="warning">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel
{
	color : red;
}</string>
    </property>
    <property name="text">
     <string>Restart the program for the changes to take effect.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="learning_rate">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>95</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
